backend/index.ts

```diff
@@ -17,29 +20,40 @@ const port = 3000;
 const ip = process.env.IP_ADDRESS ?? 'localhost';
 
 // middleware
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(bodyParser.json());
+// app.use(bodyParser.urlencoded({ extended: true }));
+// app.use(bodyParser.json());
```

Comment:
I don't think everything needs to be form data, just maybe things that require uploading a file at the same time with multipart form. Basically I want to remove bodyParser because the core Express package now includes the json body parsing and urlendoded() middleware. The other routes should work fine, the create organization route I added just uses a regular json body for now not form data.